src/console_op/data.go

```diff
@@ -377,3 +387,23 @@ func (dm DataManager) doGetPodReplicaCount(w http.ResponseWriter, r *http.Reques
 	resp.Replicas = nodeRepCount
 	SendResponseJSON(w, http.StatusOK, resp)
 }
+
+func (dm DataManager) doGetCurrentTargets(w http.ResponseWriter, r *http.Request) {
+	// only allow 'GET' calls
+	if r.Method != http.MethodGet {
+		w.Header().Set("Allow", "GET")
+		sendJSONError(w, http.StatusMethodNotAllowed,
+			fmt.Sprintf("(%s) Not Allowed", r.Method))
+		return
+	}
+
+	// Transfer the current values
+	// NOTE - not thread safe, but should be ok
+	var resp GetCurrentTargetsResponse
+	resp.TargetNumMtnNodes = numMtnNodesPerPod
```

Comment:
I don't know Go enough to understand how these variables are shared, so I don't know where this data comes from.

Reply:
It's a global. With Go anything in the same module can see everything. Kind of C'ish.